src/findings/findingsTreeDataProvider.ts

```diff
@@ -312,8 +312,11 @@ export class FindingsTreeDataProvider implements vscode.TreeDataProvider<Finding
     ));
   }
 
-  getHotspotsForFile(fileUri: string): FindingNode[] {
-    return this.findingsCache.get(fileUri)?.filter(finding => finding.findingType === FindingType.SecurityHotspot) || [];
+  getHotspotsAndTaintsForFile(fileUri: string): FindingNode[] {
+    return this.findingsCache.get(fileUri)
+      ?.filter(finding => finding.findingType === FindingType.SecurityHotspot
+        || finding.findingType === FindingType.TaintVulnerability)
```

Comment:
I believe it would be more readable to extract the predicate to a function, here I find the double `||` misleading.